### DISCUSSION TITLE
Install Lists

### DISCUSSION THREAD

**Discussion by trymeouteh** [ScoopInstaller/Scoop#6070] [Ideas]
The ability to create, import, and export a text file which will contain a list of apps.

Users can easily modify this list within scoop CLI and can export the list into a file. This list can also easily be imported into scoop CLI as well. A setting will be available to “Install apps from list” which will go into a menu were you can view all the apps on the list and choose which ones to install and which ones to ignore (not install) and then scoop CLI will install all the apps on the list that are not installed on the device all at once.

This will make it easier to get all of your apps installed on a new computer instead of searching for each app and installing each app one by one.

**Comment by niheaven** [comment]
`scoop export` and `scoop import`
